src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_combination.html.twig

```diff
@@ -35,7 +35,19 @@
         </div>
     </td>
     <td class="attribute-finalprice text-sm-right">
-      <span data-price="{{ form.vars.value.final_price }}" data-uniqid="{{ form.vars.value.id_product_attribute }}">{{ form.vars.value.final_price }}</span> {{ default_currency }}
+      <div>
+        <span class="final-price" data-price="{{ form.vars.value.final_price }}" data-uniqid="{{ form.vars.value.id_product_attribute }}">{{ form.vars.value.final_price }}</span> {{ default_currency }}
+      </div>
+      {% if 'PS_USE_ECOTAX'|configuration != 0 %}
+        {% set attributeEcotax = form.vars.value.attribute_ecotax %}
+        {% if attributeEcotax == 0 %}
```

Comment:
This could be done in php, not in the view?

Reply:
Maybe indeed, but it would mean adding some "parsing" of the form data all the time just for this.. it might be more complex for not much more value Besides this part is only used for display and as a reminder